server/src/main/java/org/apache/iotdb/db/engine/cache/ChunkCacheMetrics.java

```diff
@@ -29,7 +29,8 @@
 import java.util.Objects;
 
 public class ChunkCacheMetrics implements IMetricSet {
-  private ChunkCache chunkCache;
+
+  private final ChunkCache chunkCache;
```

Comment:
better move this class to mpp.metric package

Reply:
fixed